homeassistant/components/iammeter/const.py

```diff
@@ -0,0 +1,219 @@
+"""Constants for the Iammeter integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+
+from homeassistant.components.sensor import SensorEntityDescription, SensorStateClass
+from homeassistant.const import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfFrequency,
+    UnitOfPower,
+)
+
+DOMAIN = "iammeter"
+
+# Default config for iammeter.
+DEFAULT_IP = "192.168.2.15"
+DEFAULT_NAME = "IamMeter"
+DEVICE_3080 = "WEM3080"
+DEVICE_3080T = "WEM3080T"
+DEVICE_TYPES = [DEVICE_3080, DEVICE_3080T]
+
+
+@dataclass
+class IammeterSensorEntityDescription(SensorEntityDescription):
+    """Describes Iammeter sensor entity."""
+
+    value: Callable[[float | int], float] | Callable[[datetime], datetime] | None = None
+
+
+SENSOR_TYPES_3080: tuple[IammeterSensorEntityDescription, ...] = (
+    IammeterSensorEntityDescription(
+        key="Voltage",
+        name="Voltage",
```

Comment:
Name can be removed once the device class is set, so we rely on the translations provided by the device class.